example/weather/services/tester/run_tests.go

```diff
@@ -77,6 +77,24 @@ func recoverFromTestPanic(ctx context.Context, testName string, testCollection *
 	}
 }
 
+// Filters a testMap based on a test name that is a glob string
+// using standard wildcards https://tldp.org/LDP/GNU-Linux-Tools-Summary/html/x11655.htm
+func matchTestFilter(ctx context.Context, test string, testMap map[string]func(context.Context, *TestCollection)) (bool, []func(context.Context, *TestCollection)) {
+	match := false
+	testMatches := []func(context.Context, *TestCollection){}
+	var g glob.Glob
+	g = glob.MustCompile(test)
+	i := 0
+	for testName, _ := range testMap {
+		match = g.Match(testName)
+		if match {
+			testMatches = append(testMatches, testMap[testName])
+		}
+		i++
+	}
+	return match, testMatches
```

Comment:
I don't understand the need for returning a boolean, the caller can just check whether the returned slice is empty no?

Reply:
Yeah, good call. I was originally just returning a bool then realized I needed the matches and just appended to the return instead of replaced. I'll fix.